Dockerfile

```diff
@@ -29,7 +28,6 @@ ADD hack/installers installers
 ADD hack/tool-versions.sh .
 
 RUN ./install.sh packr-linux
-RUN ./install.sh ksonnet-linux
```

Comment:
Unfortunately, we cannot stop support ksonnet yet. Is it possible to conditionally skip it for arm?

Reply:
I have reverted the ksonnet install removal, and fix the arm64 install instead. Please check. @alexmt